content/TrueNASUpgrades/_index.md

```diff
@@ -1,29 +1,97 @@
 ---
-title: "Release Schedules"
+title: "Software Releases"
 description: "Centralized schedules and upgrade charts for software releases."
 weight: 25
 related: false
+aliases:
+ - /scale/gettingstarted/upgrades/
 ---
 
-Releases are listed by software and anticipated date.
-The type of release (Early, Stable, Maintenance) is also noted.
-The release names and dates provided here are tentative and can change at any time.
+{{< columns >}}
 
-Development timelines for [TrueNAS SCALE](#scale-schedule-timeline), [TrueNAS CORE](#core-schedule-timeline), and [TrueCommand](#truecommand-schedule-timeline) are available below.
+{{< tabbox name=scale-downloads defaultTab=4 >}}
+
+<--->
+
+{{< tabbox name=core-downloads defaultTab=4 >}}
+
+{{< /columns >}}
+
+For the latest deployment and update recommendations from iXsystems, see the [Software Status page](https://www.truenas.com/software-status/).
+
+### Nightly Builds
 
-For the latest deployment and update recommendations, see the [Software Status page](https://www.truenas.com/software-status/).
+{{< include file="_includes/NightlyTestWarning.md" >}}
 
+Nightly builds are automatically published when automated testing passes.
+**.iso** files are available to fresh install on disposable hardware or a virtual machine.
+Switching a disposable system from CORE to SCALE typically requires a fresh install using a nightly .iso file.
+
+**.update** files are also available for manually updating an existing experimental system to a nightly build.
+
+Your testing and feedback is always welcome!
+TrueNAS SCALE nightlies have an [option in the top panel](https://www.truenas.com/docs/scale/scaleuireference/toptoolbar/#how-would-you-rate-this-page?) to submit feedback directly to the development team.
+Let us know what is working well and what can be improved!
+
+### Legacy Releases
+
+Legacy TrueNAS versions are provided for historical context and upgrade pathways.
+They are provided "as-is" and typically do not receive further maintenance releases.
+Individual releases are within each major version.
+
+Legacy releases can only be used by downloading the .iso file and freshly installing to the hardware.
+See the [Documentation Archive](https://www.truenas.com/docs/archive/) for content related to these releases.
+
+## Upgrade Paths
 
 {{< columns >}}
+**TrueNAS SCALE**
 
-{{< releaselist name=core-releases defaultTab=2 >}}
+{{< mermaid class="mermaid_sizing" >}}
```

Comment:
Text on the mermaid charts is really small now that the sidebar toc has the content area squished down